exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporter.java

```diff
@@ -92,6 +95,12 @@ private void validate(final ElasticsearchExporterConfiguration configuration) {
               "Elasticsearch prefix must not contain underscore. Current value: %s",
               configuration.index.prefix));
     }
+
+    if (configuration.bulk.memoryLimit > RECOMMENDED_MAX_BULK_MEMORY_LIMIT) {
```

Comment:
(Optional) Maybe also add a lower boundary; in case someone enters 10 in the configuration thinking it will be 10 MB

Reply:
That would be an option :+1:  But I think it is not necessary because a lower value doesn't break anything. It is also documented and has the default value in the templates.